VERSION_UPDATE_GUIDE.md

```
@@ -0,0 +1,233 @@
+
+# Find version-specific references
+grep -r "4\.[12][0-9]\|stable-4\.[12][0-9]" --include="*.yaml" telco-*
+
+# Find ClusterInstance examples
+grep -r "^kind: ClusterInstance"
+```
+
+## Testing After Updates
+
+1. Verify no stale version references remain:
+   ```bash
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)